### PR TITLE
[py][build] Re-add Windows to CI workflows

### LINKED ISSUE #16390: [🐛 Bug]: [py][build] Test errors when run with Bazel on  Windows (long paths)
### Description

Several Python tests fail when running with Bazel on Windows. They pass with Bazel on Linux, and also pass on Windows when run directly with pytest instead of Bazel. I get a `cannot shorten the path enough` error (see below).

This seems to be caused by how Bazel handles long paths on Windows: https://github.com/bazelbuild/bazel/pull/22532

### Reproducible Code

```shell
bazel test //py:common-chrome-bidi-test/selenium/webdriver/common/executing_async_javascript_tests.py
```

### Debugging Logs

```logs
ERROR(tools/test/windows/tw.cc:1302) ERROR: src/main/native/windows/process.cc(95): WaitableProcess::Create(D:\_bazel\execroot\_main\bazel-out\x64_windows-fastbuild\bin\py\common-chrome-bidi-test\selenium\webdriver\common\executing_async_javascript_tests.py.exe.runfiles\_main\py\common-chrome-bidi-test\selenium\webdriver\common\executing_async_javascript_tests.py.exe): ERROR: src/main/native/windows/util.cc(292): AsExecutablePathForCreateProcess(D:\_bazel\execroot\_main\bazel-out\x64_windows-fastbuild\bin\py\common-chrome-bidi-test\selenium\webdriver\common\executing_async_javascript_tests.py.exe.runfiles\_main\py\common-chrome-bidi-test\selenium\webdriver\common\executing_async_javascript_tests.py.exe): ERROR: src/main/native/windows/util.cc(262): GetShortPathNameW(\\?\D:\_bazel\execroot\_main\bazel-out\x64_windows-fastbuild\bin\py\common-chrome-bidi-test\selenium\webdriver\common\executing_async_javascript_tests.py.exe.runfiles\_main\py\common-chrome-bidi-test\selenium\webdriver\common\executing_async_javascript_tests.py.exe): cannot shorten the path enough
```

### PR DESCRIPTION
**Pull request by cgoldberg**
### 🔗 Related Issues

Fixes #16390

### 💥 What does this PR do?

This PR adds a new GitHub Actions workflow jobs for Python: `browser-tests-windows`

This runs the Python integration tests on Windows/Chrome and Windows/Edge. The job is identical to the existing `browser-tests` job, but it includes the `fsutil 8dot3name set 0` command to workaround the long path name issue that Bazel has on Windows. 

Note: There's probably a better way to do this using a single matrix job where the command is only run when the runner is using Windows, but I couldn't figure that out.

### 💡 Additional Considerations

- Unit tests on Windows are still not included because some tests randomly timeout on the GHA runners (can't reproduce locally)
- Firefox on Windows is not included because it is insanely slow and would increase the job run time to 2+ hours

### 🔄 Types of changes

- Build/Test

___

### **PR Type**
Tests

___

### **Description**
- Add Windows CI job for browser tests

- Include Windows path workaround with `fsutil 8dot3name set 0`

- Support Chrome and Edge on Windows

- Minor formatting improvements to existing commands

___

<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Tests</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>ci-python.yml</strong><dd><code>Add Windows CI jobs and formatting</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

.github/workflows/ci-python.yml

<ul><li> Add <code>browser-tests-windows</code> job for Chrome, Edge<br> <li> Include <code>fsutil 8dot3name set 0</code> command for path issues<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/16396/files#diff-a7ec86144013d78c9f1d734aa65a898e66b609638aa7b872491ee3858ebebbeb">+56/-6</a>&nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Formatting</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>new_session_tests.py</strong><dd><code>Remove extra blank line</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

py/test/unit/selenium/webdriver/remote/new_session_tests.py

- Remove extra blank line after license header

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/16396/files#diff-796ccf287d5fe4869871971128599eb2c753a904cc8181399272687dea26441e">+0/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>subtyping_tests.py</strong><dd><code>Remove extra blank line</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

py/test/unit/selenium/webdriver/remote/subtyping_tests.py

- Remove extra blank line after license header

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/16396/files#diff-3dfa911f886315f3a6e37f89d8a97910a56c2131cdbaa44f151d333c0549c097">+0/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

</details>

___

### CODE REVIEW

File: .github/workflows/ci-python.yml [hunk 1]
```diff
@@ -121,6 +146,6 @@ jobs:
       name: Integration Tests (${{ matrix.browser }}, ${{ matrix.os }})
       browser: ${{ matrix.browser }}
       os: ${{ matrix.os }}
-      cache-key: py-browser-${{ matrix.browser }}
+      cache-key: py-browser-macos-${{ matrix.browser }}
```
Comment (p0deje) [inline .github/workflows/ci-python.yml]: This is not needed because `setup-bazel` automatically adds OS to cache key.

File: .github/workflows/ci-python.yml [hunk 2]
```diff
@@ -107,7 +111,28 @@ jobs:
...
+        include:
+          - browser: chrome
+            os: windows
+          - browser: edge
+            os: windows
+    with:
+      name: Integration Tests (${{ matrix.browser }}, ${{ matrix.os }})
+      browser: ${{ matrix.browser }}
+      os: ${{ matrix.os }}
+      cache-key: py-browser-windows-${{ matrix.browser }}
```
Comment (p0deje) [inline .github/workflows/ci-python.yml]:
```suggestion
      cache-key: py-browser-${{ matrix.browser }}
```

**Review by p0deje — APPROVED** [review]

File: .github/workflows/ci-python.yml (hunk 1 above)
Comment (cgoldberg) [reply .github/workflows/ci-python.yml]: fixed

File: .github/workflows/ci-python.yml (hunk 2 above)
Comment (cgoldberg) [reply .github/workflows/ci-python.yml]: fixed
